examples/enable-operator-and-auto-instrumentation/otel-demo-nodejs.md

```diff
@@ -20,15 +20,11 @@ curl https://raw.githubusercontent.com/signalfx/splunk-otel-collector-chart/main
 #### 2.1 Deploy the Helm Chart with the Operator enabled
 
 To install the chart with operator in an existing cluster, make sure you have cert-manager installed and available.
-Both the cert-manager and operator are subcharts of this chart and can be enabled with `--set operatorcrds.install=true,operator.enabled=true,certmanager.enabled=true`.
+Both the cert-manager and operator are subcharts of this chart and can be enabled with `--set operatorcrds.install=true,operator.enabled=true`.
```

Comment:
```suggestion
The opentelemetry-operator is a subchart of this chart and can be enabled with `--set operatorcrds.install=true,operator.enabled=true`.
```

Not sure why are we mentioning the cert-manager

Reply:
Yeah my grep missed these in our example demo docs for this specific section. Pushed out a commit to update these docs with your input in mind.